helix-term/src/keymap.rs

```diff
@@ -291,6 +300,14 @@ pub fn default() -> Keymaps {
                 code: KeyCode::Esc,
                 modifiers: KeyModifiers::NONE
             } => commands::exit_select_mode as Command,
+            KeyEvent {
+                code: KeyCode::Home,
+                modifiers: KeyModifiers::NONE
+            } => commands::move_line_start,
```

Comment:
Need to use appropriate extend methods here as well

Reply:
Can you add these extend mappings for `^` and `$` too?

Reply:
Say no more ~ on it